sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -257,7 +252,7 @@ export interface ManifestPageResponse extends Array<ArtifactManifestProperties>
 // @public
 export interface OciAnnotations extends Record<string, unknown> {
     authors?: string;
-    createdOn?: Date;
+    created?: string;
```

Comment:
Curious - why move away from strong types?

Reply:
Same reasoning as below -- this is what comes back from `JSON.parse`.